README.md

```diff
@@ -12,9 +12,9 @@ Note: Offline mode is removed in this fork, a server is required. You need to st
 
 ### Features
 
-* Support normal blocks, plants (grass, flowers, etc.) and transparency (glass).
-* Day / night cycles and a textured sky dome.
-* No height limit (65536 blocks really).
+* Supports normal blocks, plants (grass, flowers, etc.) and transparency (glass).
+* Day / night cycles and a textured skybox.
+* No height limit (technically, just 65536 blocks, but you won't reach this I bet :P).
```

Comment:
Actually, I think we manage to remove this artificial limit some commits ago. The client is now loading a 3d sphere of chunks around it, regardless in which dimensions, so there is actually no height limit any more. I'll update it in another pull at some point :smile: